### PR TITLE
Update restraint and cki_lib environment part 2 of 2

### PR DESCRIPTION
**Pull request by ftjahjad**
Updated cki beaker tests including sub-tests:
networking  pciutils          sound      storage    test     tracepoints
packages    power-management  rt-tests   standards  stress   trace  vm

### CODE REVIEW

**Comment by ftjahjad** [comment]
@cki-bot , test please

**Comment by ftjahjad** [comment]
@emanchado and @rasibley  , seeing failed on networking tests,  I am looking on it
and seeing that I remove accidentally rhtsrequires that needed for dependencies,  going back to restore it. 
Work in Progress

**Comment by mh21** [comment]
@cki-bot , test please to see whether I broke the github bot 🤔

**Comment by ftjahjad** [comment]
@cki-bot , please test again

File: networking/common/include.sh [hunk 1]
```diff
@@ -5,7 +5,6 @@ NETWORK_COMMONLIB_DIR=$(dirname $(readlink -f $BASH_SOURCE))
 networkLib=$NETWORK_COMMONLIB_DIR
 
 # include beaker default environmnet
-. /usr/bin/rhts_environment.sh
 . /usr/share/beakerlib/beakerlib.sh || . /usr/lib/beakerlib/beakerlib.sh
```
Comment (rasibley) [inline networking/common/include.sh]: Shouldn't this be `. /usr/share/beakerlib/beakerlib.sh || exit 1`

**Comment by rasibley** [comment]
@ftjahjad have you confirmed it works by testing it out by removing restraint-rhts ?

File: networking/common/include.sh (hunk 1 above)
Comment (ftjahjad) [reply networking/common/include.sh]: will update with exit 1 on next push including sync-up with master

**Comment by ftjahjad** [comment]
Hi @rasibley 
beaker without restraint (removing restraint-rhts with distribution command workaround). it seems working, 
rhel8 https://beaker.engineering.redhat.com/jobs/4157409
rhel7 https://beaker.engineering.redhat.com/jobs/4157404

**Comment by ftjahjad** [comment]
@rasibley , please take a look on my PR update.

**Comment by rasibley** [comment]
@cki-bot please test

File: networking/firewall/000infralib/Makefile [hunk 2]
```diff
@@ -83,6 +83,3 @@ $(METADATA): Makefile
 	@echo "RhtsRequires: beakerlib" >> $(METADATA)
```
Comment (rasibley) [inline networking/firewall/000infralib/Makefile]: @ftjahjad can you change this to repoRequires ?

File: networking/route/pmtu/Makefile [hunk 3]
```diff
@@ -86,7 +86,3 @@ $(METADATA): Makefile
 	@echo "RhtsRequires: kernel-kernel-networking-common" >> $(METADATA)
```
Comment (rasibley) [inline networking/route/pmtu/Makefile]: same here

File: packages/httpd/mod_ssl-smoke/Makefile [hunk 4]
```diff
@@ -60,9 +60,8 @@ $(METADATA): Makefile
 	@echo "Requires:        python2 python3" >> $(METADATA)
 	@echo "Requires:        python3-lxml" >> $(METADATA)
 	@echo "RhtsRequires:    library(httpd/http)" >> $(METADATA)
```
Comment (rasibley) [inline packages/httpd/mod_ssl-smoke/Makefile]: and here

File: packages/httpd/php-mysql-sanity/Makefile [hunk 5]
```diff
@@ -61,9 +61,8 @@ $(METADATA): Makefile
 	@echo "RhtsRequires:    library(mysql/basic)" >> $(METADATA)
```
Comment (rasibley) [inline packages/httpd/php-mysql-sanity/Makefile]: And here :-) and everywhere else please!

**Comment by rasibley** [comment]
Hi @ftjahjad I spoke with the Restraint Maintainer and it we can only use RepoRequires if the library is already open sourced and in the same repo, otherwise it appears many of the tests are pulling in internal beaker task libraries so we'll need to review these and make sure they're all open sourced before moving to metadata. In the mean time I think we can go ahead and merge, I will just review the cki-bot results once more before doing so.

**Comment by ftjahjad** [comment]
Hi @rasibley ,
Thanks for  checking to restraint maintainer.

I am going to start working  a PR for opening internal library , and we can remove RhtsRequires with restraint repoRequires. when it is done, the next PR will be converting to restraint metadata :-D

**Pull request merged**